frontend/src/components/create-user-page/CreateUser.tsx

```diff
@@ -1,3 +1,118 @@
-import React from "react";
+import React, { FormEvent, useContext, useState } from "react";
+import { LoginContext } from "../login/LoginManager";
 
-export const CreateUser: React.FunctionComponent = () => <></>;
+export const CreateUser: React.FunctionComponent = () => {
+  const loginContext = useContext(LoginContext);
+
+  const [formData, setFormData] = useState({
+    name: "",
+    username: "",
+    email: "",
+    password: "",
+  });
+  const validatePassword = (passwordToCheck: string) => {
+    const upper = /(.*[A-Z].*)/;
+    const lower = /(.*[a-z].*)/;
+    const number = /[0-9]/;
+    const special = /[-+_!@#$%^&*.,?]/;
+
+    if (passwordToCheck.length < 8) {
+      alert("Please make sure password is longer than 8 characters.");
+      return false;
+    }
+
+    if (!upper.test(passwordToCheck)) {
+      alert("Please make sure Password Includes an UpperCase character");
+      return false;
+    }
+
+    if (!lower.test(passwordToCheck)) {
+      alert("Please make sure Password Includes a LowerCase character");
+      return false;
+    }
+
+    if (!number.test(passwordToCheck)) {
+      alert("Please make sure Password Includes a Digit");
+      return false;
+    }
+
+    if (!special.test(passwordToCheck)) {
+      alert(
+        "Please make sure Password Includes a special character from -+_!@#$%^&*.,?"
+      );
+      return false;
+    }
```

Comment:
Using `alert`s is fine for now, but isn't the best UX (pop-up windows are very jarring!)

In your follow-up styling ticket, could you also make sure to replace these with, for example, red text that appears below the fields that aren't correct?

Reply:
Ok. But may need some help with this.

Reply:
We will resolve this when we do styling ticket 60